source/configuration_manual/authentication/dict.rst

```diff
@@ -85,16 +129,6 @@ Dict configuration
     mail = maildir:%{dict:userdb.home}/Maildir
```

Comment:
Can userdb.home be used without userdb_objects being specified? Clarify how exactly it works and probably use a new section.